staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -315,6 +321,8 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	ValidatingAdmissionPolicy: {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.32
 
+	WatchFromCacheWithoutResourceVersion: {Default: false, PreRelease: featuregate.Beta},
```

Comment:
Normally we name features things that start false and progress to true (defaulting to false in 1.27+ for now and changing to true in the future as the underlying issues get resolved).

We _want_ to watch from cache in 1.27+ and watch from storage in the future as the underlying issues get resolved, right?

That implies a name like `WatchFromStorageWithoutResourceVersion` defaulting to false for now and maybe true in the future?